openc3/lib/openc3/tools/table_manager/table.rb

```diff
@@ -44,9 +44,6 @@ class Table < Packet
     # Constructor for a TableDefinition
     def initialize(name, endianness, type, description, filename)
       super(TARGET, name, endianness, description, '', TableItem)
-      # ONE_DIMENSIONAL and TWO_DIMENSIONAL are deprecated so translate
```

Comment:
I think we should remove documentation on ONE_DIMENSTIONAL/TWO_DIMENSIONAL (so all text references), but it doesn't hurt to leave the keywords and not break COSMOS 4 table definitions.